ts/parts-more/RadialAxis.ts

```diff
@@ -340,18 +341,28 @@ radialAxisMixin = {
     getLinePath: function (
         this: Highcharts.RadialAxis,
         lineWidth: number,
-        radius?: number
+        radius?: number,
+        innerRadius?: number
     ): Highcharts.RadialAxisPath {
         var center = this.center,
             end,
             chart = this.chart,
             r = pick(radius, center[2] / 2 - this.offset),
             path: Highcharts.RadialAxisPath;
 
+        if (innerRadius === void 0) {
```

Comment:
Just nitpicking: everywhere else we check `typeof variable === 'undefined'`

Reply:
Corrected, thanks!